flash/audio/speech_recognition/data.py

```diff
@@ -48,7 +48,7 @@ def deserialize(self, sample: Any) -> Dict:
         encoded_with_padding = (sample + "===").encode("ascii")
         audio = base64.b64decode(encoded_with_padding)
         buffer = io.BytesIO(audio)
-        data, sampling_rate = sf.read(buffer)
+        data, sampling_rate = librosa.load(buffer, sr=16000)
```

Comment:
can we expose the sample rate here at all?

Reply:
Done :smiley: